src/grid/Grid.tsx

```diff
@@ -362,7 +362,7 @@ export function Grid<
   // Because we're using react-spring in imperative mode, we're responsible for
   // firing animations manually whenever the tiles array updates
   useEffect(() => {
-    springRef.start();
+    void springRef.start();
```

Comment:
Should the returned Promises be caught?
```suggestion
    springRef.start().forEach((p) => void p.catch(logger.error));
```